docs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "test/common.sh"
+
+build_kb
```

Comment:
we do not need to do those calls.
Whe we can add those via the Makefile right?

Reply:
Hi @camilamacedo86 I have a question here:

I can understand that if we run `make generate`, the `build_kb` will be loaded and kubebuilder would be built.

What I'm thinking is, if it's necessary to have a separate cmd, that can individually generate docs based on the code?

What we have now is `make generate-docs`, that does not include `build_kb`.